MinervaBot.py

```diff
@@ -28,40 +51,71 @@
 login = driver.find_element_by_link_text('Login')
 login.send_keys(Keys.RETURN)
 
-# Acha as entradas para o id e senha
-idcpf = driver.find_element_by_name('bor_id')
-password = driver.find_element_by_name('bor_verification')
+# Le os dados para login
+try:
+    inputcpf, inputpassword = ler_dados('login.txt')
+    salvar = True
+except FileNotFoundError:
+    print("Falha ao encontrar o arquivo de login")
+    inputcpf, inputpassword, salvar = configurar_dados(None)
+except IndexError:
+    print("Arquivo de login incompleto")
+    inputcpf, inputpassword, salvar = configurar_dados(True)
+
+# Entra em loop ate obter sucesso no login
+while True:
+    # Acha as entradas para o id e senha
+    idcpf = driver.find_element_by_name('bor_id')
+    password = driver.find_element_by_name('bor_verification')
 
-# Preenche as entradas com os parametros dados
-idcpf.send_keys(inputcpf)
-password.send_keys(inputpassword, Keys.RETURN)
+    # Preenche as entradas com os parametros dados
+    idcpf.clear()
```

Comment:
I get it should be a good idea to always clear the text entry element before inserting the values, but why clearing only `idcpf` and not `password`?

Reply:
When login fails, the `idcpf` field is filled with the value input on previous page, but `password` remains empty.

Reply:
@brunelli oh, I see, that's great! Everything with the login improvements looks good to me, I'm merging the PR.